Sources/ArcGISToolkit/Resources/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,7 @@
+"Could not access the certificate file." = "Could not access the certificate file.";
+
+"The certificate file was invalid." = "The certificate file was invalid.";
+
+"The password was invalid." = "The password was invalid.";
+
+"The certificate file or password was invalid." = "The certificate file or password was invalid.";
```

Comment:
This file isn't needed. It would only be needed if the keys aren't the same as the values.

Reply:
The exposing string for localization is not working for the toolkit package. See below error. We'll need this file to request the localized strings from localization team. I would suggest keeping this file until we figure out the export issue. I can remove the file if you still feel strongly.  

![Screen Shot 2023-01-23 at 2 58 08 PM](https://user-images.githubusercontent.com/4164028/214171283-8c0e7c85-189a-48c7-82aa-56fe069de827.png)

Reply:
I'm fine with leaving it, but I can foresee it easily becoming out-of-date (new localized string added in code but not in the strings file). 

That localization failure appears to be an Xcode bug. If you look at the build logs, it appears to be building for an unsupported platform (macOS). We should probably submit a feedback report to Apple about that.

Reply:
In fact, their file is already incomplete. It is missing the title of the certificate error alert, the button titles, etc.

Reply:
Yes, I've logged an issue for it.
https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/issues/247

Reply:
> That localization failure appears to be an Xcode bug. If you look at the build logs, it appears to be building for an unsupported platform (macOS). We should probably submit a feedback report to Apple about that.

We'll have to create a repro project to log an issue. I'll work on it.